lib/origami-build-tools-cli.js

```diff
@@ -96,6 +96,8 @@ if (tasks.isValid(argument)) {
 				taskResult
 					.on('error', reportTaskError)
 					.on('end', reportFinished);
+				// Make sure it reaches 'end'
+				taskResult.resume();
```

Comment:
To be defensive, you might want to add to the condition on line 95: `if (taskResult.on && taskResult.resume)`, we should probably also handle the case where we don't understand the returned value from the task, but that can be in another PR.